test/python/python_passes.py

```diff
@@ -5,58 +5,318 @@
+        %tile_0_1 = AIE.tile(0, 1)
+        %tile_0_2 = AIE.tile(0, 2)
+        %tile_0_3 = AIE.tile(0, 3)
+        %tile_1_0 = AIE.tile(1, 0)
+        %tile_1_1 = AIE.tile(1, 1)
+        %tile_1_3 = AIE.tile(1, 3)
+        %tile_2_0 = AIE.tile(2, 0)
+        %tile_3_0 = AIE.tile(3, 0)
+        %tile_2_2 = AIE.tile(2, 2)
+        %tile_3_1 = AIE.tile(3, 1)
+        %tile_5_0 = AIE.tile(5, 0)
+        %tile_5_2 = AIE.tile(5, 2)
+        %tile_6_0 = AIE.tile(6, 0)
+        %tile_7_0 = AIE.tile(7, 0)
+        %tile_7_1 = AIE.tile(7, 1)
+        %tile_7_2 = AIE.tile(7, 2)
+        %tile_7_3 = AIE.tile(7, 3)
+        %tile_8_0 = AIE.tile(8, 0)
+        %tile_8_1 = AIE.tile(8, 1)
+        %tile_8_2 = AIE.tile(8, 2)
+        %tile_8_3 = AIE.tile(8, 3)
+        
+        AIE.flow(%tile_5_2, DMA : 0, %tile_0_1, DMA : 0)
+        AIE.flow(%tile_5_2, DMA : 0, %tile_0_2, DMA : 0)
+        AIE.flow(%tile_5_2, DMA : 0, %tile_0_3, DMA : 0)
+        AIE.flow(%tile_5_2, DMA : 0, %tile_8_2, DMA : 0)
+        
+        AIE.flow(%tile_8_0, DMA : 0, %tile_0_0, DMA : 0)
+        AIE.flow(%tile_8_0, DMA : 0, %tile_8_3, DMA : 0)
       }
-    }
     """
 
-    # CHECK: AIE.tile
-    # CHECK: AIE.buffer
-    # CHECK: arith.constant
-    # CHECK: arith.constant
-    # CHECK: memref.store
-    # CHECK: AIE.end
-    # CHECK: AIE.core
-    # CHECK: builtin.module
-    with Context() as ctx, Location.unknown():
-        aie.dialects.aie.register_dialect(ctx)
-        _aie_python_passes.register_python_pass_demo_pass(print_ops)
-        mlir_module = Module.parse(module)
-        PassManager.parse("builtin.module(python-pass-demo)").run(mlir_module.operation)
-
+    register_pathfinder_flows_with_python(find_paths)
+    mlir_module = Module.parse(module)
+    device = mlir_module.body.operations[0]
+    pm = PassManager.parse("AIE.device(aie-create-pathfinder-flows-with-python)")
+    # pm.enable_ir_printing()
+    pm.run(device.operation)
 
-testPythonPassDemo()
+    # %switchbox_0_1 = AIE.switchbox(%tile_0_1) {
+    #   AIE.connect<East : 0, DMA : 0>
+    # }
+    # %switchbox_0_2 = AIE.switchbox(%tile_0_2) {
+    #   AIE.connect<East : 0, DMA : 0>
+    # }
+    # %switchbox_0_3 = AIE.switchbox(%tile_0_3) {
+    #   AIE.connect<East : 0, DMA : 0>
+    # }
+    # %switchbox_1_1 = AIE.switchbox(%tile_1_1) {
+    #   AIE.connect<East : 0, West : 0>
+    # }
+    # %tile_1_2 = AIE.tile(1, 2)
+    # %switchbox_1_2 = AIE.switchbox(%tile_1_2) {
+    #   AIE.connect<East : 0, West : 0>
+    # }
+    # %switchbox_1_3 = AIE.switchbox(%tile_1_3) {
+    #   AIE.connect<East : 0, West : 0>
+    # }
+    # %tile_2_1 = AIE.tile(2, 1)
+    # %switchbox_2_1 = AIE.switchbox(%tile_2_1) {
+    #   AIE.connect<East : 0, West : 0>
+    # }
+    # %switchbox_2_2 = AIE.switchbox(%tile_2_2) {
+    #   AIE.connect<East : 0, West : 0>
+    # }
+    # %tile_2_3 = AIE.tile(2, 3)
+    # %switchbox_2_3 = AIE.switchbox(%tile_2_3) {
+    #   AIE.connect<East : 0, West : 0>
+    # }
```

Comment:
Are these (and the ones below) supposed to be `# CHECK:` directives?

Reply:
yup (whoops)